WSL/about.md

```diff
@@ -1,20 +1,18 @@
 ---
 title: What is Windows Subsystem for Linux 
-description: Learn about the Windows Subsystem for Linux, including the different versions and ways you can use them.
-keywords: BashOnWindows, bash, wsl, windows, windowssubsystem, gnu, linux
-ms.date: 09/27/2021
+description: Learn about the Windows Subsystem for Linux, including the different versions and ways you can use them. Microsoft Loves Linux.
+ms.date: 11/20/2023
 ms.topic: article
-ms.assetid: 3cefe0db-7616-4848-a2b6-9296746a178b
 ---
 
 # What is the Windows Subsystem for Linux?
 
-The Windows Subsystem for Linux lets developers run a GNU/Linux environment -- including most command-line tools, utilities, and applications -- directly on Windows, unmodified, without the overhead of a traditional virtual machine or dualboot setup.
+Windows Subsystem for Linux (WSL) is a feature of Windows that allows you to run a Linux environment on your Windows machine, without the need for a separate virtual machine or dual booting.  WSL is designed to provide a seamless and productive experience for developers who want to use both Windows and Linux at the same time.
```

Comment:
```suggestion
Windows Subsystem for Linux (WSL) is a feature of Windows that allows you to run a heavily integrated Linux environment on your Windows machine.  WSL is designed to provide a seamless and productive experience for developers who want to use both Windows and Linux at the same time.
```
Adjusted wording since we do use a virtual machine for WSL